unlock-app/package.json

```diff
@@ -82,6 +82,7 @@
     "set-link:darwin:freebsd:linux:sunos": "pushd src && (test -e artifacts && rm -f artifacts) && ln -s ../../smart-contracts/build artifacts",
     "set-link:win32": "cd src && if exist artifacts (rmdir artifacts /q /s || del artifacts) && cmd /c mklink /d artifacts ..\\..\\smart-contracts\\build",
     "build": "next build src",
+    "build-paywall": "NODE_ENV=production rollup -c rollup.paywall.config.js -o ./src/static/paywall.min.js",
```

Comment:
Are we only building this in production?

Reply:
Yes, the minimized paywall is built using production settings because it generates a minimized file. I think it will be important as a next step to create a Dev build with sourcemaps as a next step. Currently the paywall is so simple that it is not absolutely necessary for v0. As soon as it starts being the paywall instead of simply loading the iframe for it then yes, we will want to be able to debug it :). That's my take, what do you think?

And to be explicit, there is nothing in this PR that would make it difficult to add on a Dev build. Thanks for pushing on this, it's an important point!!